generic-styles/content-baked.less

```diff
@@ -0,0 +1,598 @@
+// Styles for the contents of a page.
+// This should **only** be imported inside a containing selector like:
+//
+//     #content { @import 'this_file.less'; }
+
+@import "./_make-block.less";
+
+
+#main-content {
+
+  // Override the defaults so this matches webview
+  max-width: 96rem;
+  margin: auto;
+
+  @import "./_content-common.less";
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    margin: 1.5rem 0 1rem 0;
+    font-weight: bold;
+  }
+
+  h3 + section {
+    margin-top: 0;
+  }
+
+  ul, ol {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    color: @gray;
+  }
+
+  .stepwise {
+    list-style-type: none;
+    padding-left: 0;
+    > li {
+      display: flex;
+      .os-stepwise-token {
+        white-space: pre;
+      }
+      .os-stepwise-content {
+        > ul, ol {
+          padding-left: 1rem;
+        }
+      }
+    }
+  }
+
+  .os-stepwise {
+    list-style-type: none;
+    padding-left: 0;
+    > li {
+      display: flex;
+      .os-stepwise-token {
+        white-space: pre;
+      }
+      .os-stepwise-content {
+        > ul, ol {
+          padding-left: 1rem;
+        }
+      }
+    }
+  }
+
+  .circled {
+    list-style-type: none;
+    padding-left: 1rem;
+  }
+
+  [data-bullet-style="none"] {
+    list-style-type: none;
+  }
+
+  iframe {
+    display: block;
+    margin: 3rem auto;
+  }
+
+  .centered-text {
+    display: block;
+    font-weight: normal;
+    text-align: center;
+  }
+
+  .os-chapter-outline {
+    ul {
+      margin-top: 0;
+    }
+  }
+
+  .colored-text {
+    color: #CC0000;
+  }
+
+  .no-emphasis {
+    font-weight: normal; //to ensure that all terms with this tag will display unbolded
+  }
+
+  // It is already displayed at the top of the page in blue
+  // the `>` is because collated titles (like Key Concepts) contain descendent
+  // document-title elements.
+  [data-type="page"],
+  [data-type="composite-page"] {
+    > [data-type="document-title"] {
+      display: none;
```

Comment:
are these not removed by the regex?

Reply:
They probably are; I just copied the styles over; we don't really report CSS coverage but I'll remove.